docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -202,37 +201,34 @@ Typically, you need to back-fill the recording rules when you are using a long r
 and you upgraded to Scylla Monitoring 3.8 about three months ago.
 
 If you open the Overview dashboard and look at your entire retention time (in our example 1 year) you will see that while most of the graphs do
-show the data, the latency graphs have a missing period, in our example - from the entire year, the latency graph will only show the last three months.
-
-That nine months gap (12 months minus 3) is what we want to fill with back-filling.
+display the data, the latency graphs are missing a period of time.  The latency graph will only show the last three months in our example from the entire year. That nine months gap (12 months minus 3) is what we want to fill with back-filling.
 
 The point in time that the graphs start will be your back-filling end time. Check in the graph for the exact time.
 
 Backfilling Process
 -------------------
-backup
+Backup
 ^^^^^^
-If you have a long retention period you are using an external directory that holds the Prometheus data, back it up, in case
-If you have a long retention period, you are using an external directory that holds the Prometheus data back it up; if something goes wrong in the process, you can revert the process.
+Backup the external directory containing Prometheus data; if something goes wrong, you can revert the changes.
+
+To complete the process, you must restart Monitoring Stack at least once. You cannot complete the process without providing the path to the external directory with Prometheus data using the ``-d`` command line option. 
 
-To complete the process you will need to restart the monitoring stack at least once. If you are not using an external directory (The ``-d``
-command-line option) You cannot complete it.
 
 Restart the monitoring stack
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-You need to stop the monitoring stack and run the ``stat-all.sh`` command with an additional flag:
+You need to stop the monitoring stack and run the ``start-all.sh`` command with an additional flag (The 365d retention time used here is used only as an example):
 
-``-b "--storage.tsdb.allow-overlapping-blocks"``
+``start-all.sh -d data_dir -b "--storage.tsdb.allow-overlapping-blocks --storage.tsdb.retention.time=365d"``
 
 Create the data files
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-We will use the Promtool utility; it's already installed for you if you are using the docker container. 
-You will need the start time and end time for the process, in our example the start time is 360 days ago and the end time is 90 days ago.
+We will create the data files using the Promtool utility, which has been installed in the Docker container. To run the utility, you must pass the start time and end time in the epoch format. The following example shows one of the ways to convert the times to epoch when the start time is 360 and the end time is 90 days ago:
```

Comment:
I would make it clearer, it's insalled in the prometheus docker container

Reply:
"We will create the data files using the Promtool utility, which is installed in the **Prometheus** Docker container". - Is that what you mean?

Reply:
Hello @annastuchlik / @amnonh  , the paragraph could be:

We will create the data files using the Promtool utility, which is installed in the Prometheus Docker container. To run the utility, you must pass the start time and end time in the epoch format. The following example shows one of the ways to convert the times to epoch when the start time is 360 and the end time is 90 days ago:

Reply:
Sounds good to me.

Reply:
Hello @annastuchlik , great, thanks for feedback.